### DISCUSSION TITLE
Auto centering of my model in UVtools

### DISCUSSION THREAD

**Discussion by Strix786** [sn4k3/UVtools#790] [Q&A] [answered]
Hi everyone,

I'm encountering a problem in UVtools while trying to position a model for 3D printing. My aim is to place the model on one side of the build plate. In the UVtools interface, the model seems to move to the desired location correctly. However, when I start the printing process using the file generated by UVtools, the model ends up being positioned in the center of the build plate.

To describe the issue in more detail: In the UVtools window, there's a green rectangle that surrounds my model. It appears that this rectangle, and its contents, get centered on the build plate regardless of where I place the model. Adding a second model to the scene creates a scenario where the green rectangle encompasses both models, which are positioned apart (one in the center and the other at the top left corner). But, when I print, both models are centered on the build plate while maintaining their relative positions to each other.

Has anyone faced a similar issue or can provide insights on how to resolve this?

Thanks in advance for your help!

**Comment by sn4k3** [comment]
What is your printer?

If you load the printed file back in UVtools is the location of the model correct/your desire?

**Comment by Strix786** [comment]
My printer is a Phrozen Mighty 8K.

Yes, UVtools display the file with the model in the correct position. 
![image](https://github.com/sn4k3/UVtools/assets/151196187/a37a85d7-c487-434c-acd2-b3c8036df35a)

I think that if I could get the green rectangle to occupy the totality of the space, it would work as intended.

**Reply by sn4k3** [reply, marked as answer]
I'm not aware of any auto-center function by the printer itself but try to slice a file and print directly without UVtools, ex: 20x20x20mm cube and print it at an edge, see if it comes centered after all.

By default UVtools does not move objects and always show the real slice image, if it is to be moved by UVtools you would see the image on the new position.

**Comment by Strix786** [comment]
Well, I don't know where I messed up, but everything is working as expected now. 

Thank you for your help!
